packages/core/src/internal/coreEffectRunners/cancelled.js

```diff
@@ -0,0 +1,3 @@
+export default function runCancelledEffect(env, effectPayload, cb, { mainTask }) {
+  cb(Boolean(mainTask._isCancelled))
```

Comment:
no needs to cast to Boolean, it always Boolean https://github.com/redux-saga/redux-saga/blob/master/packages/core/src/internal/proc.js#L169

Reply:
got it.